api/src/migrations/add-national_admin-role.js

```diff
@@ -1,59 +1,72 @@
 const _ = require('underscore'),
+  request = require('request'),
+  url = require('url'),
+  series = require('async/series'),
   { promisify } = require('util'),
-  db = require('../db-nano'),
-  logger = require('../logger'),
-  series = require('async/series');
+  environment = require('../environment'),
+  logger = require('../logger');
 
 const DEFAULT_STRUCTURE = {
   names: [],
   roles: [],
 };
 
-const addRole = (dbname, role, callback) =>
-  db.request(
-    {
-      db: dbname,
-      method: 'GET',
-      path: '_security',
+const addRole = (dbname, role, callback) => {
+  request.get({
+    url: url.format({
+      protocol: environment.protocol,
+      hostname: environment.host,
+      port: environment.port,
+      pathname: `${dbname}/_security`,
+    }),
+    auth: {
+      user: environment.username,
+      pass: environment.password
     },
-    (err, result) => {
-      if (err) {
-        return callback(err);
-      }
-
-      // In CouchDB 1.x, if you have not written to the _security object before
-      // it is empty.
-      if (!result.admins) {
-        result.admins = DEFAULT_STRUCTURE;
-      }
+    json: true
+  }, (err, result) => {
+    if (err) {
+      return callback(err);
+    }
 
-      if (!result.admins.roles.includes(role)) {
-        logger.info(`Adding ${role} role to ${dbname} admins`);
-        result.admins.roles.push(role);
-      }
+    // In CouchDB 1.x, if you have not written to the _security object before
+    // it is empty.
+    if (!result.admins) {
```

Comment:
the `request.get` callback receives 3 parameters: `err`, `response` and `body` (in this order: https://github.com/request/request#super-simple-to-use). The `response` object (which you are using here) looks like { statusCode: x, headers: [...], body: {...}, request: { ... } }. 
This is the reason why the `routing` e2e test failed, because the `_security` object looked like:
```
{
  "statusCode": 200,
  "body": {
    "admins": {
      "names": [],
      "roles": []
    }
  },
  "headers": {
    "cache-control": "must-revalidate",
    "connection": "close",
    "content-length": "51",
    "content-type": "application/json",
    "date": "Tue, 20 Nov 2018 09:48:00 GMT",
    "server": "CouchDB/2.2.0 (Erlang OTP/19)",
    "x-couch-request-id": "3ba961521b",
    "x-couchdb-body-time": "0"
  },
  "request": {
    "uri": {
      "protocol": "http:",
      "slashes": true,
      "auth": null,
      "host": "localhost:5984",
      "port": "5984",
      "hostname": "localhost",
      "hash": null,
      "search": null,
      "query": null,
      "pathname": "/medic/_security",
      "path": "/medic/_security",
      "href": "http://localhost:5984/medic/_security"
    },
    "method": "GET", 
    "headers": {
      "authorization": "Basic YWRtaW46cGFzcw==",
      "accept": "application/json"
    }
  },
  "admins": {
      "names": [],
      "roles": [
        "national_admin"
      ]
    }
}
```

instead of just 

```
{
  "admins": {
      "names": [],
      "roles": [
        "national_admin"
      ]
   }
}
```